src/App.tsx

```diff
@@ -1,155 +1,120 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { TodosList } from './components/TodosList';
+import { Error } from './components/Error';
+import { Footer } from './components/Footer';
+import { FilterBy } from './utils/Enums/FilterBy';
+import { Header } from './components/Header';
+import { ErrorType } from './utils/Enums/ErrorType';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { FilterByType } from './types/FilterBy';
+
+import { getTodos, post } from './api/todos';
+
+const USER_ID = 10217;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
```

Comment:
No clear why default state is null instead of empty array
```suggestion
  const [todos, setTodos] = useState<Todo[]>([]);
```